src/cloudsc_gpu/dwarf_cloudsc_gpu.F90

```diff
@@ -167,6 +171,32 @@ PROGRAM DWARF_CLOUDSC
      & )
 #endif
 
+#ifdef CLOUDSC_GPU_SCC_CUF_K_CACHING
+print '(1X,A46)', 'Executing CLOUDSC-GPU, "SCC-CUF K-CACHING" CUDA variant...'
```

Comment:
Minor detail but the "A64" trims the string after `CUD` instead of `CUDA variant`. Might need to increase the string length.